book/content/credit/credit.md

```diff
@@ -61,6 +61,10 @@ Showing an example reference in the most common referencing format in your disci
 
 If you use GitHub, GitLab or similar, consider creating a `CITATION` file in each repository containing an example reference.
 
+### Make your research identifiable
+
+As part of the citation for your software and data it is important to give people a persistent link to it, this will usually be a DOI. DOIs (or more formally Digital Object Identifiers) are unique, persistent identifiers which are attached to a digital object such as a journal article, dataset or piece of software. Using DOIs makes it much easier for others to cite your data, reduces the risk of linkrot and means you can track how your research is being used and cited.
```

Comment:
```suggestion
As part of the citation for your software and data it is important to give people a persistent link to it, this will usually be a DOI.
DOIs (or more formally Digital Object Identifiers) are unique, persistent identifiers which are attached to a digital object such as a journal article, dataset or piece of software.
Using DOIs makes it much easier for others to cite your data, reduces the risk of linkrot and means you can track how your research is being used and cited.
```